src/DataManagement/Saving/TheadingCalculator.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using ePiggyWeb.DataManagement.Entries;
+using ePiggyWeb.DataManagement.Goals;
+using ePiggyWeb.Utilities;
+
+namespace ePiggyWeb.DataManagement.Saving
+{
+    public class ThreadingCalculator
```

Comment:
As I understand this is the class which is supposed to work with the threads? Calling the alternateSavingsCalculator? Because then  alternateSavingsCalculator could be static since even if there's 3 threads they will be working with different lists.

Reply:
What's the advantage to using static, anyway, could you remind/tell me? At least at the moment it doesn't seem like an issue to me

Reply:
Also, will they be working with different lists? :D i'm not sure about that. probably safer to just leave it like that, no?

Reply:
I think we can leave it as it is